chatkit-core/src/main/kotlin/com/pusher/chatkit/CurrentUser.kt

```diff
@@ -162,6 +270,13 @@ class CurrentUser(
             callback: (Subscription) -> Unit
     ) = makeSingleCallback({ syncCurrentUser.subscribeToRoom(roomId, messageLimit, consumer) }, callback)
 
+    /**
+     * Subscribe to a [room] for multipart messages. You need to configure the [listeners] to listen
+     * for the events you're interested in, for more information on what events are possible see the [docs][https://pusher.com/docs/chatkit/reference/android#chat-events]
+     * You can configure a [messageLimit] which returns the 10 messages that were most recently sent to the room,
```

Comment:
>  can configure a [messageLimit] which returns the 10 messages

It would return the `messageLimit` most recent messages with a default of 10 if it's not provided - this suggests it's always 10